### PR TITLE
VORTEX-5732 Infinity layer data count

### PR DESCRIPTION
**Pull request by roddyaj**

### CODE REVIEW

File: open-sphere-plugins/infinity/src/main/java/io/opensphere/infinity/envoy/InfinityEnvoy.java [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+            bool.setMust(innerMust);
+            must[0] = new Any("bool", bool);
+        }
+        else
+        {
+            must[0] = new Any("range", new Any(parameters.getTimeField(), new TimeRange(parameters.getTimeSpan())));
+        }
+        if (parameters.getGeometryType() == GeometryType.POINT)
+        {
+            must[1] = new Any("geo_bounding_box",
```
Comment (jkae) [inline open-sphere-plugins/infinity/src/main/java/io/opensphere/infinity/envoy/InfinityEnvoy.java]: Picking a random line but maybe pull these strings into constants.

File: open-sphere-plugins/infinity/src/main/java/io/opensphere/infinity/envoy/InfinityQuerier.java [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+     * @param binField the bin field
+     * @return the search response
+     * @throws QueryException if something goes wrong with the query
+     */
+    public SearchResponse query(DataTypeInfo dataType, Polygon polygon, TimeSpan timeSpan, String binField) throws QueryException
+    {
+        SearchResponse response = null;
+
+        String url = InfinityUtilities.getUrl(dataType);
+        String geomField = InfinityUtilities.getTagValue(".es-geopoint", dataType);
```
Comment (jkae) [inline open-sphere-plugins/infinity/src/main/java/io/opensphere/infinity/envoy/InfinityQuerier.java]: constant fields again. maybe a class/enum for them

File: open-sphere-plugins/infinity/src/main/java/io/opensphere/infinity/util/InfinityUtilities.java [hunk 3]
```diff
@@ -13,7 +14,39 @@
...
+
+    /**
+     * Gets the URL for the data type.
+     *
+     * @param dataType the data type
+     * @return the URL, or null
+     */
+    public static String getUrl(DataTypeInfo dataType)
+    {
+        String url = getTagValue(".es-url", dataType);
```
Comment (jkae) [inline open-sphere-plugins/infinity/src/main/java/io/opensphere/infinity/util/InfinityUtilities.java]: ^

File: open-sphere-plugins/infinity/src/main/java/io/opensphere/infinity/envoy/InfinityEnvoy.java (hunk 1 above)
Comment (roddyaj) [reply open-sphere-plugins/infinity/src/main/java/io/opensphere/infinity/envoy/InfinityEnvoy.java]: I actually prefer to have these here - it seems easier to update if the API ever changes

File: open-sphere-plugins/infinity/src/main/java/io/opensphere/infinity/envoy/InfinityQuerier.java (hunk 2 above)
Comment (roddyaj) [reply open-sphere-plugins/infinity/src/main/java/io/opensphere/infinity/envoy/InfinityQuerier.java]: Made constants

File: open-sphere-plugins/infinity/src/main/java/io/opensphere/infinity/util/InfinityUtilities.java (hunk 3 above)
Comment (roddyaj) [reply open-sphere-plugins/infinity/src/main/java/io/opensphere/infinity/util/InfinityUtilities.java]: Made constants

**Pull request merged**